src/client/App.jsx

```diff
@@ -34,16 +34,69 @@ function App() {
    * */
 
   const handleRegister = async ({ username, password }) => {
+    const option  = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({username, password})
+      };
+      try{
+        const data  = await fetch(`${apiUrl}/register`, option)
+        if(data){
+          const fetchData = await data.json();
+        }else {
+          return 'error occurred when posting'
+
+        }
+
+      }catch(e) {
+        console.log('Error in registration : ', e)
+
+      }
+
+
 
   };
 
   const handleLogin = async ({ username, password }) => {
+    const option = {
+      method: 'Post',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({username, password})
+    }
+    const logUser = await fetch (`${apiUrl}/user/login`, option)
+    if (logUser){
+      const userToken = await logUser.json()
```

Comment:
You aren`t storing the token in localstorage. So when you retrieve it it is null